client/src/components/ImageScrubber/ImageScrubber.js

```diff
@@ -149,9 +131,8 @@ const mapState = state => {
 const mapDispatch = (dispatch) => ({
   getTreesWithImagesAsync: ({ page, rowsPerPage, order, orderBy }) => dispatch.trees.getTreesWithImagesAsync({ page: page, rowsPerPage: rowsPerPage, order: order, orderBy: orderBy }),
   getLocationName: (id, lat, lon) => dispatch.imageScrubber.getLocationName({ id: id, latitude: lat, longitude: lon }),
-  toggleTreeActive: (id) => dispatch.imageScrubber.toggleTreeActive(id),
```

Comment:
This is vestigeous right now, but may come up if we want to show the tree name, though it's fine to remove from now, since we can theoretically duplicate it from the treetable view.

Reply:
@deepwinter The toggle actions were on the card itself which was a duplicate of the code in `TreeImageCard.js`. Moved the toggle actions into the `TreeImageCard`. To reuse that existing file since the `tree` object is being used to build the image card.

Reply:
it is not a toggle, it is two different properties. one sets active to false and the other sets approved to true